apps/hash-graph/tests/integration/postgres/mod.rs

```diff
@@ -4,9 +4,8 @@ mod entity_type;
 mod links;
 mod property_type;
 
-use std::{borrow::Cow, ops::Bound, str::FromStr, time::SystemTime};
+use std::{borrow::Cow, str::FromStr};
```

Comment:
I think I removed `ops::Bound` several times now, but somehow it keeps appearing. Maye it's now gone forever 😆